### PR TITLE
Augment Mood feature for Pet

### LINKED ISSUE #53: I want to get a visual cue from my pet to prompt me to do work
So that I can be motivated to work when my productivity is low

### PR DESCRIPTION
**Pull request by FyonnOh**
- Create a PetManager class to keep track of the duration after each done task. If duration exceeds 24 hours, Pet would turn hangry.
- This system works regardless of the the status of the program (ie. it works even when program is not running)

### CODE REVIEW

File: src/main/java/seedu/address/logic/PetManager.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    }
+
+    public void setPet(Pet pet) {
+        this.pet = pet;
+        this.lastDoneTaskTime = LocalDateTime.parse(pet.getLastDoneTaskTime());
+        // use this for ACTUAL
+        // this.timeForHangry = lastDoneTaskTime.plusHours(24);
+        // use this for TESTING
+        this.timeForHangry = lastDoneTaskTime.plusMinutes(1);
+        System.out.println("When log in,  TIME FOR HANGRY: " + timeForHangry);
```
Comment (BransonNg) [inline src/main/java/seedu/address/logic/PetManager.java]: remove debug prints

File: src/main/java/seedu/address/logic/PetManager.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        this.hasStarted = false;
+    }
+
+    public void setPet(Pet pet) {
+        this.pet = pet;
+        this.lastDoneTaskTime = LocalDateTime.parse(pet.getLastDoneTaskTime());
+        // use this for ACTUAL
+        // this.timeForHangry = lastDoneTaskTime.plusHours(24);
+        // use this for TESTING
+        this.timeForHangry = lastDoneTaskTime.plusMinutes(1);
```
Comment (BransonNg) [inline src/main/java/seedu/address/logic/PetManager.java:50]: shouldn't this be plusDays?

**Review by BransonNg — APPROVED** [review]
LGTM! Addded comments previously already

File: src/main/java/seedu/address/logic/PetManager.java (hunk 2 above)
Comment (FyonnOh) [reply src/main/java/seedu/address/logic/PetManager.java:50]: In the comments, there are the codes for testing and actual. I did not comment out appropriately, will take note in the future!

File: src/main/java/seedu/address/logic/PetManager.java (hunk 1 above)
Comment (FyonnOh) [reply src/main/java/seedu/address/logic/PetManager.java]: Alright!

**Pull request merged**